src/SoftwareSerial.h

```diff
@@ -32,11 +32,76 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 // If only one tx or rx wanted then use this as parameter for the unused pin
 constexpr int SW_SERIAL_UNUSED_PIN = -1;
 
-enum SoftwareSerialConfig {
-	SWSERIAL_5N1 = 0,
-	SWSERIAL_6N1,
-	SWSERIAL_7N1,
-	SWSERIAL_8N1,
+// Options for SoftwareSerialConfig
+constexpr uint8_t SWSERIAL_NB_BIT_MASK       = 0B00011000;
```

Comment:
Really just detail and my personal preference, but anyway, how about SWSERIAL_BITS_MASK, or SWSERIAL_BITCNT_MASK, I don't feel good about "NB"

Reply:
I borrowed the setup from uart.h but I'm not particularly fond of it. Renaming these will probably be my biggest hurdle :-)
Anyway, shortened to DB, PB and SB for data, parity and stop bits. Far from perfect but at least shorter.